ykman-gui/qml/OtpChalRespView.qml

```diff
@@ -51,41 +51,11 @@ ColumnLayout {
             }
 
             BreadCrumbRow {
-                BreadCrumb {
-                    text: qsTr("Home")
-                    action: views.home
-                }
-
-                BreadCrumbSeparator {
-                }
-                BreadCrumb {
-                    text: qsTr("OTP")
-                    action: views.otp
-                }
-
-                BreadCrumbSeparator {
-                }
-                BreadCrumb {
-                    text: qsTr(SlotUtils.slotNameCapitalized(
-                                   views.selectedSlot))
-                    action: views.otp
-                }
-
-                BreadCrumbSeparator {
-                }
-
-                BreadCrumb {
-                    text: qsTr("Select Credential Type")
-                    action: views.pop
-                }
-
-                BreadCrumbSeparator {
-                }
-
-                BreadCrumb {
-                    text: qsTr("Challenge-response")
-                    active: true
-                }
+                items: [
+                    { text: qsTr("OTP") },
+                    { text: qsTr(SlotUtils.slotNameCapitalized(views.selectedSlot)) },
```

Comment:
I think the middle one is dangerous, I get `qrc:/qml/OtpYubiOtpView.qml:69: Error: qsTr(): first argument (sourceText) must be a string` in the logs after configuring a Yubico OTP.

Reply:
Yeah, I saw that too. There's also

```
qrc:/qml/BreadCrumbRow.qml:25:23: Unable to assign [undefined] to QString
```

which seems related. It looks to me like `views.selectedSlot` evaluates to undefined or some other out-of-domain value during the transition between views, or something like that.

Reply:
Fixed by 6bf4aa7 and e5cbb56